docs/week-3.md

```diff
@@ -0,0 +1,568 @@
+wrapped in a `<div>`. This is because React requires us to return a contained component. This means that there must be
+a type of container that has what we want to display housed inside. In most cases using a `<div>` is the best way to 
+go. However, you can use any piece of JSX as a wrapper/container.
+
+So, we have covered that a component is just a piece of a React App. A cool thing we can do in React is that we can
+use components that we have made in other components that we make.
+
+### Using Components
+
+Remember our`LuckyNum` component we made above? We are going to use this component in our Profile component below in
+order to display your favorite number along with your name and favorite color.
+
+```
+import React from 'react';
+import LuckyNum from './LuckyNum';
+
+/* 
+   Notice how there is no return statement. If we are only 
+   returning JSX (no javascript inside of our component) 
+   we can use () to wrap our component
+*/
+
+const Profile = () => (
+
+    <div>
+        <img src="profileImage.jpg"/>
+        <h1>Hi, my name is John! My favorite color is blue.</h1>
+        <LuckyNum />
+    </div>
+);
+
+export default Profile;
+```
+
+As you can see we import the component `LuckyNum` from the typescript file we wrote. Once the component is imported
+we are able to use it in our new component. The syntax to do this is by creating a tag to indicate where we
+want the imported component to be placed.
+
+## Props
+
+Looks like things are going well. We have successfully displayed John's profile picture, name, favorite color and
+lucky number. Now John's friend Doug wants to display his lucky number on his profile. John is happy to help and
+gives him the `LuckyNum` component we made. However, Doug's lucky number is not the same as John's so Doug cannot
+use the `LuckyNum` component! John has a solution: use props.
+
+### What are Props and why use them?
+
+Props are a nice way of making your components reusable. Instead of hard coding all of our data and information
+into our components, we can pass in props to give some malleability to our components. A great way to understand
+props is to compare them to how functions work. For example, let us print a message to the console.
+
+```
+function printHi() {
+
+    console.log("hi");
+
+}
+```
+
+We successfully printed "hi" to the console! Now what if we wanted to print "hey" to the console. We would have
+to make a new function because `printHi()` only prints "hi". We would then have to make `printHey()`.
+
+```
+function printHey() {
+
+    console.log("hey");
+
+}
+```
+
+We have now printed "hi" and "hey," but it is very repetitive and not efficient. If we wanted to print another
+message, we would have to make a new function OR we can just add a parameter to a function to print whatever we
+want to the console and we would not need to make a new function each time. All we would need to do is pass in
```

Comment:
I would remove this last sentence since you already said pretty much the same thing